ssl/ssl_sess.c

```diff
@@ -794,6 +808,9 @@ int SSL_set_session(SSL *s, SSL_SESSION *session)
     if (session != NULL) {
         SSL_SESSION_up_ref(session);
         s->verify_result = session->verify_result;
+        s->hit = 1;
+    } else {
+        s->hit = 0;
```

Comment:
Why is the change in this function necessary?

Reply:
Not needed, it's not in any our branches; so it must be cruft.